From foo@baz Tue Aug 17 07:42:17 PDT 2010
Date: Tue, 17 Aug 2010 07:42:17 -0700
To: Greg KH <gregkh@suse.de>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: kabi: fix up struct transacion_s abi change

This changed in the 2.6.32.17 kernel release, and as it's just a
bitfield, we can safely hide it from the kabi checker to keep things
building properly.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/jbd2.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/include/linux/jbd2.h
+++ b/include/linux/jbd2.h
@@ -653,8 +653,9 @@ struct transaction_s
 	 * waiting for it to finish.
 	 */
 	unsigned int t_synchronous_commit:1;
+#ifndef __GENKSYMS__
 	unsigned int t_flushed_data_blocks:1;
-
+#endif
 	/*
 	 * For use by the filesystem to store fs-specific data
 	 * structures associated with the transaction
